BoxContentSDK/BoxContentSDK/Protocols/BOXContentCacheClientProtocol.h

```diff
@@ -213,6 +213,9 @@
                               withFolder:(BOXFolder *)folder
                                    error:(NSError *)error;
 
+- (void)deleteChildrenOfFolderWithID:(NSString *)folderID
```

Comment:
Should it be associated with a request? This feels odd of place, can we see if it should conform more to the existing pattern or whether it belongs somewhere else?